site/src/pug/includes/body.pug

```diff
@@ -6,7 +6,7 @@ include ./fork.pug
 
         .heading.flex
             h1 devSwag
-            p 😎 swag opportunities for developers
+                p 😎 swag opportunities for developers
```

Comment:
With this extra tabulation, the `p` tag goes into the `h1` one. This is a prohibited behavior and should be changed.

References: 
- https://www.w3.org/TR/2011/WD-html5-author-20110705/the-h1-h2-h3-h4-h5-and-h6-elements.html
- https://www.w3.org/TR/2011/WD-html5-author-20110705/content-models.html#phrasing-content

Reply:
That wasn't supposed to happen 🙈